app/frontend/src/components/forms/PrintOptions.vue

```diff
@@ -0,0 +1,251 @@
+
+          <p>You can either print the page from the browser, or upload a Template file to have a structured PDF version of this submission.</p>
+          <div v-if="showTemplateUpload">
+            <v-file-input
+              counter
+              :clearable="true"
+              label="Upload template file"
+              persistent-hint
+              prepend-icon="attachment"
+              required
+              mandatory
+              show-size
+              v-model="templateForm.files"
+            />
+            <v-card-actions>
+              <v-spacer />
+              <v-tooltip top>
+                <template v-slot:activator="{ on }">
+                  <v-btn
+                    color="primary"
+                    class="btn-file-input-submit"
+                    :disabled="!templateForm.files"
+                    id="file-input-submit"
+                    :loading="loading"
+                    @click="generate"
+                    v-on="on"
+                  >
+                    <v-icon :left="$vuetify.breakpoint.smAndUp">save</v-icon>
+                    <span v-if="$vuetify.breakpoint.smAndUp">Submit</span>
+                  </v-btn>
+                </template>
+                <span>Submit to CDOGS and Download</span>
+              </v-tooltip>
+            </v-card-actions>
+          </div>
+        </v-card-text>
+
+        <v-card-actions class="justify-center">
+          <v-btn class="mb-5 mr-5" color="primary" @click="printBrowser">
+            <span>Print via Browser</span>
+          </v-btn>
+          <v-btn
+            class="mb-5 mr-5"
+            color="primary"
+            @click="showTemplateUpload = !showTemplateUpload"
+          >
+            <span>Use Template</span>
+          </v-btn>
+          <v-btn class="mb-5" outlined @click="dialog = false">
+            <span>Cancel</span>
+          </v-btn>
+        </v-card-actions>
+      </v-card>
+    </v-dialog>
+  </span>
+</template>
+
+<script>
+import { mapActions, mapGetters } from 'vuex';
+import formService from '@/services/formService';
+import { NotificationTypes } from '@/utils/constants';
+
+export default {
+  data() {
+    return {
+      dialog: false,
+      showTemplateUpload: false,
+      exportFormat: 'csv',
+      submissionRecord: {},
+      validFileInput: null,
+      loading: false,
+      templateForm: {
+        contexts: '{}',
+        contextFiles: null,
+        convertToPDF: null,
+        files: null,
+        templateContent: '',
+        contentFileType: null,
+        outputFileName: '',
+        outputFileType: null,
+      },
+    };
+  },
+  props: {
+    submissionId: String,
+  },
+  computed: {
+    ...mapGetters('form', ['form']),
+    files() {
+      return this.templateForm.files;
+    },
+  },
+  methods: {
+    ...mapActions('notifications', ['addNotification']),
+    async printBrowser() {
+      this.dialog = false;
+      setTimeout(() => {
+        window.print();
+      }, 100);
```

Comment:
I'd recommend putting a quick comment here as to why this delay is needed so that future developers know not to remove it.